cmd/srrs/main.go

```diff
@@ -26,6 +26,7 @@ var (
 	httpAddr = flag.String("http", defaultAddr, "HTTP service address")
 	static   = flag.String("static", "", "Path to the static assets")
 	sock     = flag.String("socket", filepath.Join(os.TempDir(), "trc.sock"), "Path to the unix socket")
+	useTCP   = flag.Bool("tcp", false, "Use a tcp socket instead of a Unix socket")
```

Comment:
This should not be hard-coded, it should be configurable.
Maybe it's possible to use `io.TeeReader`(see [here](https://golang.org/pkg/io/#TeeReader)) or something similar to multiplex? (so support the input from both and output on both)

Reply:
what is hardcoded here? it is a configuration on itself. default is false anyway

Reply:
The port